apps/chatbot/docker/app.test.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12.4-slim-bullseye
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+  apt-get install -y \
+  curl
+
```

Comment:
like this it does not read dynamic pages. edit with:

```
RUN apt-get update && \
  apt-get install -y \
  curl \
  wget \
  zip

RUN wget https://github.com/rphrp1985/selenium_support/raw/main/chrome_114_amd64.deb && \
  apt-get install -y ./chrome_114_amd64.deb && \
  wget https://chromedriver.storage.googleapis.com/114.0.5735.90/chromedriver_linux64.zip && \
  unzip chromedriver_linux64.zip && \
  mv chromedriver /usr/bin/chromedriver
```

Reply:
done